lib/tasks/manage.rake

```diff
@@ -38,21 +38,22 @@ namespace :manage do
           subscriber: subscriber,
           subscriber_list: source_subscriber_list
         )
+        if existing_subscription
```

Comment:
How about making this a guard clause, so `next unless existing_subscription`. This avoids indenting all the following code.

Reply:
Yes, can do that 🙂